addons/website/views/snippets/s_splash_intro.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_splash_intro" name="Splash Intro">
+    <section class="s_splash_intro o_full_screen_height oe_img_bg o_bg_img_center o_bg_img_origin_border_box o_colored_level" style="background-image: url('/web/image/website.s_splash_intro_default_image');">
+        <div class="o_we_bg_filter bg-black-15"/>
+        <div class="container">
+            <div class="row align-items-end">
+                <div class="col-lg-7 pt88 pb88">
+                    <p class="lead text-muted"><strong>Find the solution for you</strong></p>
+                    <h1 class="display-3">Built for Impact.<br/>Designed to Inspire.</h1>
+                    <p><br/></p>
+                    <p>
+                        <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary o_translate_inline"><t t-out="cta_btn_text">Discover More</t></a>
+                    </p>
+                </div>
+            </div>
+        </div>
+        <a class="o_scroll_button d-flex rounded-circle align-items-center justify-content-center mx-auto mb-4 o_cc o_cc1 o_not_editable" href="#" contenteditable="false" title="Scroll down to next section">
```

Comment:
No need for the `contenteditable=false` attribute, it will be added automatically thanks to the `o_not_editable` class :slightly_smiling_face: 

No need for the `d-flex` class too, `display: flex` is already set in the `o_scroll_button` CSS rule :+1: